# from array import array
# a = array('i',[1,2,3])
# try:
# 	print(a[3])
# except Exception as exception:
# 	print(exception)

a = [1,2,3]
print(a.sizeof)